lib/scripts/services/yandexTranslate.js

```diff
@@ -43,20 +47,68 @@ export class YandexTranslate {
   }
 
   /**
+   * Convert object to list
+   * @param {Object} words - words object
+   * @returns {Array} wordList - array of words
+   */
+  toList(words) {
+    var wordList = [];
+    for (let word in words) {
+      wordList.push(word);
+    }
+    return wordList;
+  }
+
+  /**
+   * Breaks the words list into 128 words and returns
```

Comment:
I am sorry, I took this function from `googleTranslate` and forgot to edit that. Will correct it.